microshift_cli_ref/microshift-cli-overview.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="cli-tools-overview"]
+= {product-title} CLI tools
+include::_attributes/attributes-microshift.adoc[]
+:context: cli-tools-overview
+
+toc::[]
+
+A user builds, deploys, and manages applications, as well as managing clusters while working with {product-title}.
+
+{product-title} can use different command-line interface (CLI) tools that simplify these tasks by enabling users to perform various administration and development operations from the terminal.
+These tools expose simple commands to manage the deployments, as well as interact with each component of the system.
+
+In addition to built-in `microshift` command types and Linux CLI tools, the optional OpenShift CLI (`oc`) tool is available for you to use if you are already familiar with {OCP} and Kubernetes.
+
+The enabled subset of commands available with the `oc` tool are commonly used by {product-title} users. It helps both cluster administrators and developers to perform operations across {product-title} using the terminal.
+
+[role="_additional-resources"]
+.Additional resources
+
+. link:https://access.redhat.com/documentation/en-us/microshift/4.12/html/microshift_cli_ref/microshift-cli-using-oc.adoc#microshift-cli-oc-about[Installing the `oc` tool for MicroShift].
+
+. link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/cli_tools/openshift-cli-oc[OpenShift CLI (oc)]: A full description of `oc` as provided by the {OCP} documentation. Commands focused on multi-node deployments, projects, and developer tooling are not supported by {product-title}.
```

Comment:
Please refer to the [contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) on how to format Additional resources sections.